cluster/internal/data.proto

```diff
@@ -2,7 +2,8 @@ package internal;
 
 message WriteShardRequest {
     required uint64 ShardID = 1;
-    repeated Point Points = 2;
+    required uint64 OwnerID = 2;
```

Comment:
why is this part of the request object? The only reason for the protobuf is for serialization over the wire. By the time it gets to that stage, it's already sent to the owner. Seems like unnecessary bloat to me.